pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -94,8 +94,8 @@ let
 
   srcs = {
     x86_64-linux = fetchurl {
-      url = "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz";
-      sha256 = "16zk7d0sxm2j00vb002mjj38wxcxxlahnfdb9lmkmkfms9p9xfkb";
+      url = "https://www.torproject.org/dist/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz";
```

Comment:
That redirects to dist.torproject.org.

Reply:
ohh I did not know. But any way When you run the code it actually works instead of giving a 404

Reply:
You'll notice that if you go to this link https://dist.torproject.org/torbrowser/ the latest stable is 10.5.3, but that directory only holds android versions. For whatever reason this link 
https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz
does not work but this one
https://www.torproject.org/dist/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz
does work. The link that works was taken from the tor projects downloads page. I would think that the tor project is encouraging us to use the link provided there.

Reply:
Both links work

https://dist.torproject.org/torbrowser/10.5.2/tor-browser-linux64-10.5.2_en-US.tar.xz
https://www.torproject.org/dist/torbrowser/10.5.2/tor-browser-linux64-10.5.2_en-US.tar.xz

Reply:
This is what I get when I try to install it. Its the same thing when I replace "tor-browser-bundle-bin" with "torbrowser"

!!Note where it says "curl: (22) The requested URL returned error: 404"

nix-shell -p tor-browser-bundle-bin
these derivations will be built:
  /nix/store/vikjzv2kbjh8kd9z413zwqpx3qipcjn5-tor-browser-linux64-10.0.18_en-US.tar.xz.drv
  /nix/store/pfsvdpya436169hm4fpzj7dymwpgq9nk-tor-browser-bundle-bin-10.0.18.drv
building '/nix/store/vikjzv2kbjh8kd9z413zwqpx3qipcjn5-tor-browser-linux64-10.0.18_en-US.tar.xz.drv'...

trying https://dist.torproject.org/torbrowser/10.0.18/tor-browser-linux64-10.0.18_en-US.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   266    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download tor-browser-linux64-10.0.18_en-US.tar.xz from any mirror
builder for '/nix/store/vikjzv2kbjh8kd9z413zwqpx3qipcjn5-tor-browser-linux64-10.0.18_en-US.tar.xz.drv' failed with exit code 1
cannot build derivation '/nix/store/pfsvdpya436169hm4fpzj7dymwpgq9nk-tor-browser-bundle-bin-10.0.18.drv': 1 dependencies couldn't be built
error: build of '/nix/store/pfsvdpya436169hm4fpzj7dymwpgq9nk-tor-browser-bundle-bin-10.0.18.drv' failed


But when I use this

nix-shell -p torbrowser -I nixpkgs=https://github.com/PulellaP/nixpkgs/archive/patch-1.tar.gz

It builds and I can run the tor-browser command in the shell and the torbrowser opens up.